tsl/src/nodes/decompress_chunk/planner.h

```diff
@@ -7,7 +7,28 @@
 
 #include <postgres.h>
 
+typedef enum
```

Comment:
In other enums (e.g., `CompressionColumnType`, `MinMaxResult`, `ScanTupleResult`) we capitalize the enum members. Is there a specific reason for these two enums not following this approach?

Reply:
I don't know, all-caps is kind of inconvenient to type :) I think all-caps is some remnant of ancient C where there were no enums, and these constants were defines. Postgres uses both styles.